src/main/java/gregicadditions/recipes/MachineCraftingRecipes.java

```diff
@@ -261,105 +158,148 @@ public static void init() {
+		}
+		if(GAConfig.GT5U.highTierAutoclaves) {
+			registerMachineRecipe(GATileEntities.AUTOCLAVE, "IGI", "IMI", "CPC", 'M', HULL, 'P', PUMP, 'C', CIRCUIT, 'I', PLATE, 'G', GLASS);
+		}
+		if(GAConfig.GT5U.highTierMixers) {
+			registerMachineRecipe(GATileEntities.MIXER, "GRG", "GEG", "CMC", 'M', HULL, 'E', MOTOR, 'R', ROTOR, 'C', CIRCUIT, 'G', GLASS);
+		}
+		if(GAConfig.GT5U.highTierLaserEngravers) {
+			registerMachineRecipe(GATileEntities.LASER_ENGRAVER, "PEP", "CMC", "WCW", 'M', HULL, 'E', EMITTER, 'P', PISTON, 'C', CIRCUIT, 'W', CABLE);
+		}
+		if(GAConfig.GT5U.highTierFormingPresses) {
+			registerMachineRecipe(GATileEntities.FORMING_PRESS, "WPW", "CMC", "WPW", 'M', HULL, 'P', PISTON, 'C', CIRCUIT, 'W', CABLE);
+		}
+		if(GAConfig.GT5U.highTierForgeHammers) {
+			registerMachineRecipe(GATileEntities.FORGE_HAMMER, "WPW", "CMC", "WAW", 'M', HULL, 'P', PISTON, 'C', CIRCUIT, 'W', CABLE, 'A', OreDictNames.craftingAnvil);
+		}
+		if(GAConfig.GT5U.highTierFluidHeaters) {
+			registerMachineRecipe(GATileEntities.FLUID_HEATER, "OGO", "PMP", "WCW", 'M', HULL, 'P', PUMP, 'O', COIL_HEATING_DOUBLE, 'C', CIRCUIT, 'W', CABLE, 'G', GLASS);
+		}
+		if(GAConfig.GT5U.highTierSifters) {
+			registerMachineRecipe(GATileEntities.SIFTER, "WFW", "PMP", "CFC", 'M', HULL, 'P', PISTON, 'F', MetaItems.ITEM_FILTER, 'C', CIRCUIT, 'W', CABLE);
+		}
+		if(GAConfig.GT5U.highTierArcFurnaces) {
+			registerMachineRecipe(GATileEntities.ARC_FURNACE, "WGW", "CMC", "PPP", 'M', HULL, 'P', PLATE, 'C', CIRCUIT, 'W', CABLE_QUAD, 'G', new UnificationEntry(OrePrefix.ingot, Materials.Graphite));
+		}
+		if(GAConfig.GT5U.highTierPlasmaArcFurnaces) {
+			registerMachineRecipe(GATileEntities.PLASMA_ARC_FURNACE, "WGW", "CMC", "TPT", 'M', HULL, 'P', PLATE, 'C', BETTER_CIRCUIT, 'W', CABLE_QUAD, 'T', PUMP, 'G', new UnificationEntry(OrePrefix.ingot, Materials.Graphite));
+		}
+		if(GAConfig.Misc.highTierCollector) {
+			registerMachineRecipe(GATileEntities.AIR_COLLECTOR, "WFW", "PHP", "WCW", 'W', Blocks.IRON_BARS, 'F', MetaItems.ITEM_FILTER, 'P', PUMP, 'H', HULL, 'C', CIRCUIT);
+		}
 		registerMachineRecipe(GATileEntities.MASS_FAB, "CFC", "QMQ", "CFC", 'M', HULL, 'Q', CABLE_QUAD, 'C', BETTER_CIRCUIT, 'F', FIELD_GENERATOR);
-		registerMachineRecipe(GATileEntities.REPLICATOR, "EFE", "CMC", "EQE", 'M', HULL, 'Q', CABLE_QUAD, 'C', BETTER_CIRCUIT, 'F', FIELD_GENERATOR, 'E', EMITTER);
-		if (GAConfig.Misc.highTierCollector) registerMachineRecipe(GATileEntities.AIR_COLLECTOR, "WFW", "PHP", "WCW", 'W', Blocks.IRON_BARS, 'F', MetaItems.ITEM_FILTER, 'P', PUMP, 'H', HULL, 'C', CIRCUIT);
+		registerMachineRecipe(GATileEntities.REPLICATOR,
+							  "EFE",
+							  "CMC",
+							  "EQE",
+							  'M', HULL,
+							  'Q', CABLE_QUAD,
+							  'C', BETTER_CIRCUIT,
+							  'F', FIELD_GENERATOR,
+							  'E', EMITTER);
```

Comment:
Seems a little inconsistent to reformat this one when the others were left on a single line.

Reply:
The formatting is in progress to adjust all the previous recipe registers to use this format.